dvc/command/add.py

```diff
@@ -83,6 +90,13 @@ def add_parser(subparsers, parent_parser):
             "This doesn't affect any DVC operations."
         ),
     )
+    parser.add_argument(
+        "-o",
+        "--output",
+        type=str,
+        metavar="<path>",
+        help="Output path for the file, if it lives outside the repo.",
+    )
```

Comment:
This will clash with `--out` being introduced in #5198

Reply:
It should be `--out`, but it won't clash otherwise. They are actually the same thing :)

Reply:
p.s. looking at this to try to understand iterative/dvc.org/pull/2101

Can someone explain what this does with an example perhaps? 

It's also not obvious to me how this is a step towards #4657 which doesn't seem to involve external data (but this does).

Thanks!

Reply:
I just noticed your answer @efiop but my questions remain because

> They are actually the same thing

`--out` in #5198 is not stand-alone.

Reply:
BTW should this maybe be a stand-alone, general option that `add` can always use (whether the target is external or not)? As disucssed in https://github.com/iterative/dvc/pull/5198#pullrequestreview-569646959 🙂